src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comment';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface PropsComment {
```

Comment:
the same as with function names, change it to [ComponentName]Props in all places